app/routes/endemics/biosecurity.js

```diff
@@ -1,51 +1,106 @@
 const Joi = require('joi')
 const { getEndemicsClaim, setEndemicsClaim } = require('../../session')
-const { biosecurity: biosecurityKey } = require('../../session/keys').endemicsClaim
+const { biosecurity: biosecurityKey } =
+  require('../../session/keys').endemicsClaim
 const raiseInvalidDataEvent = require('../../event/raise-invalid-data-event')
-const { urlPrefix, ruralPaymentsAgency, optionalPIHunt } = require('../../config')
-const { endemicsTestResults, endemicsBiosecurity, endemicsCheckAnswers, endemicsDiseaseStatus, endemicsBiosecurityException, endemicsVetRCVS, endemicsPIHunt, endemicsPIHuntRecommended, endemicsPIHuntAllAnimals } = require('../../config/routes')
+const {
+  urlPrefix,
+  ruralPaymentsAgency,
+  optionalPIHunt
+} = require('../../config')
+const {
+  endemicsTestResults,
+  endemicsBiosecurity,
+  endemicsCheckAnswers,
+  endemicsDiseaseStatus,
+  endemicsBiosecurityException,
+  endemicsVetRCVS,
+  endemicsPIHunt,
+  endemicsPIHuntRecommended,
+  endemicsPIHuntAllAnimals
+} = require('../../config/routes')
 const { livestockTypes } = require('../../constants/claim')
 const { getLivestockTypes } = require('../../lib/get-livestock-types')
 const { getTestResult } = require('../../lib/get-test-result')
 
 const pageUrl = `${urlPrefix}/${endemicsBiosecurity}`
-const isPIHuntPropValueYes = (value) => value === 'yes'
-const isPIHuntValidPositive = (isPositive, piHuntDone, piHuntAllAnimals) => optionalPIHunt.enabled ? (isPositive && piHuntDone && piHuntAllAnimals) : (isPositive && piHuntDone)
-const isPIHuntValidNegative = (isNegative, piHuntDone, piHuntRecommended, piHuntAllAnimals) => isNegative && piHuntDone && piHuntRecommended && piHuntAllAnimals
-const isPIHuntValid = (isPositive, piHuntDone, piHuntAllAnimals, piHuntRecommended, isNegative) => isPIHuntValidPositive(isPositive, piHuntDone, piHuntAllAnimals) || isPIHuntValidNegative(isNegative, piHuntDone, piHuntRecommended, piHuntAllAnimals)
+const isPIHuntValidPositive = (isPositive, piHuntDone, piHuntAllAnimals) =>
```

Comment:
```suggestion
const isPIHuntValidPositive = (isPositive, piHuntDone, piHuntAllAnimals) => isPositive && piHuntDone && (optionalPIHunt.enabled ? piHuntAllAnimals : true)
```
Dunno if it's any clearer but removes a bit of repitition, and puts the logic that always has to be true first